docs/TulipSport.md

```diff
@@ -0,0 +1,44 @@
+## GitHub Pages
+### 1.为GitHub Actions添加代码提交权限
+访问仓库的 `Settings > Actions > General`页面，找到`Workflow permissions`的设置项，将选项配置为`Read and write permissions`，支持CI将运动数据更新后提交到仓库中。
+
+### 2.提交配置变更代码
+1. 更新`./gatsby-config.js`中的`siteMetadata`节点。
+```javascript
+module.exports = {
+  siteMetadata: {
+    siteTitle: 'Running Page',
+    siteUrl: 'https://tulipsport.github.io/running_page',
+    logo: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQTtc69JxHNcmN1ETpMUX4dozAgAN6iPjWalQ&usqp=CAU',
+    description: 'Personal site and blog',
+    navLinks: [
+      {
+        name: 'HomePage',
+        url: 'http://tulipsport.com',
+      },
+    ],
+  },
+}
+```
+2. 更新GitHub CI的配置`./.github/workflows/run_data_sync.yml`中的配置为郁金香模式
+```yaml
+env:
+  # please change to your own config.
+  RUN_TYPE: tulipsport
+  ATHLETE: Runner's Name
+  TITLE: Runner Page Title
+  MIN_GRID_DISTANCE: 10 # change min distance here
+  TITLE_GRID: Over 10km Runs # also here
+  GITHUB_NAME: your_github_user_name
+  GITHUB_EMAIL: your_github_email
+```
+3. 在仓库的`Settings > Secrets and variables > Actions`页面添加名为`TULIPSPORT_TOKEN`的新secrets信息，并配置相应token（[如何获取token？](https://tulipsport.rdshoep.com)）
```

Comment:
This section is crucial, and to improve its clarity and conciseness, it would be beneficial to include 'tulip' as a type, just like [Keep](https://github.com/yihong0618/running_page/blob/master/README-CN.md#Keep)